src/actions/articleActions.js

```diff
@@ -34,3 +38,24 @@ export const postArticle = postData => dispatch => {
       toast.error(errorMessage, { autoClose: false, hideProgressBar: true });
     });
 };
+
+export const addComment = (postData, article) => dispatch => {
+  axiosInstance
+    .post(`/api/articles/${article}/comments/`, postData)
+    .then(() => {
+      dispatch(addCommentSuccess(true));
+    });
+};
+
+export const fetchComments = article => dispatch => {
+  dispatch(getCommentInititated(true));
+  axiosInstance
+    .get(`/api/articles/${article}/comments/`)
+    .then((response) => {
+      dispatch(getCommentsSuccess(response.data));
+    })
+    .catch(() => {
```

Comment:
Nice work @MuhanguziDavid.
This section might need more work though.
Why are you logging out the user?
Might it not be necessary to display that reason to that user?

Reply:
Thanks, @RonKbS , I am logging out the user because the only way in which an error can occur is if the user's authentication token has expired.
I shall add a toast to display the reason for being logged out to the user.